cmd/notation/sign.go

```diff
@@ -28,17 +28,17 @@ func signCommand(opts *signOpts) *cobra.Command {
 		opts = &signOpts{}
 	}
 	command := &cobra.Command{
-		Use:   "sign [reference]",
-		Short: "Sign OCI artifacts",
-		Long: `Sign OCI artifacts
+		Use:   "sign [flags] <reference>",
+		Short: "Sign artifacts",
+		Long: `Sign artifacts
 
 Prerequisite: a signing key needs to be configured using the command "notation key".
 
 Example - Sign a container image using the default signing key, with the default JWS envelope:
   notation sign <registry>/<repository>:<tag>
 
 Example - Sign a container image using the default signing key, with the COSE envelope:
-  notation sign --envelope-type cose <registry>/<repository>:<tag> 
+  notation sign --envelope-format cose <registry>/<repository>:<tag> 
```

Comment:
```suggestion
  notation sign --signature-format cose <registry>/<repository>:<tag> 
```

Reply:
The flag is called "--envelope-format" right? Are you suggesting renaming all --envelope-format to --signature-format?